src/base-importer/BaseImporter.js

```diff
@@ -100,10 +96,26 @@ export default class BaseImporter {
         )
     }
 
-    _generateUrl(url, queries) {
+    _extractQueryParamsFromAuth(auths) {
+        return (auths || []).filter((auth) => {
+            return auth instanceof Auth.ApiKey && auth.get('in') === 'query'
+        }).map((auth) => {
+            return new KeyValue({
+                key: encodeURI(auth.get('name') || ''),
+                value: encodeURI(auth.get('name') || '')
+            })
+        }).toArray()
+    }
+
+    _generateUrl(url, queries, auths) {
         let _url = url
-        if (queries && queries.length > 0) {
-            _url += '?' + queries.map((keyValue) => {
+
+        let queryParams = (queries || []).concat(
+            this._extractQueryParamsFromAuth(auths)
+        )
+
+        if (queryParams.length > 0) {
+            _url += '?' + queryParams.map((keyValue) => {
                 return keyValue.get('key') + '=' + (keyValue.get('value') || '')
             }).join('&')
```

Comment:
Yeah, I think I did this way because queryParams are uriEncoded directly during the parsing, while Auth params aren't. This was a way to make them catch up, but I agree that it'd be safer and clearer to have all of them be uriEncoded there